### DISCUSSION TITLE
Deploy BentoService to Kubernetes can not work

### DISCUSSION THREAD

**Discussion by zhuyaguang** [bentoml/BentoML#2819] [Q&A]
saved_path=$(bentoml get IrisClassifier:latest --print-location --quiet) this cmd does not work

![image](https://user-images.githubusercontent.com/8857976/181145422-47b64e80-5d38-45d9-8db7-e632259b8e9c.png)

**Comment by parano** [comment]
@zhuyaguang you might be looking at an outdated documentation, could you point me to where you are referring to ?

**Reply by zhuyaguang** [reply]
https://docs.bentoml.org/en/0.13-lts/deployment/kubernetes.html#prerequisites

**Reply by zhuyaguang** [reply]
i search bentoml on k8s , this page is result

**Reply by parano** [reply]
thanks, @zhuyaguang, that link was under 0.13 version, the 1.0 version has some behavior changes.

I'd recommend taking a look at Yatai https://github.com/bentoml/Yatai if you are looking to deploy BentoML on kubernetes.

**Reply by zhuyaguang** [reply]
OK
